From 0d94363bc87f107decd7bebd5ec951fe3d8a5df2 Mon Sep 17 00:00:00 2001
From: Evan Wang <xswang@marvell.com>
Date: Tue, 21 Mar 2017 15:55:32 +0800
Subject: [PATCH 1526/2241] dts: a3700: enable UTMI PHY for DB and community
 board

- For DB and community board, UTMI PHY is needed by USB2,
  then it is necessary to initialize UTMI PHY, so the
  patch enable it as default.

Change-Id: If9e01bded8c1cea1b3b958bd3cd46fe3b4dbd490
Signed-off-by: Evan Wang <xswang@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37731
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-community.dts |  8 ++++++++
 arch/arm64/boot/dts/marvell/armada-3720-db.dts        |  8 ++++++++
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi          | 16 ++++++++--------
 3 files changed, 24 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-community.dts b/arch/arm64/boot/dts/marvell/armada-3720-community.dts
index 7578073..2fe36c6 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-community.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-community.dts
@@ -123,6 +123,14 @@
 				phy-names = "usb";
 			};
 
+			utmi_usb32: utmi@5d000 {
+				status = "okay";
+			};
+
+			utmi_usb2: utmi@5f000 {
+				status = "okay";
+			};
+
 			sdhci1: sdhci@d0000 {
 				wp-inverted;
 				bus-width = <4>;
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
index be9f65a..6bb85ba 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
@@ -163,12 +163,20 @@
 				phy-names = "usb2", "usb3";
 			};
 
+			utmi_usb32: utmi@5d000 {
+				status = "okay";
+			};
+
 			usb@5e000 {
 				status = "okay";
 				phys = <&utmi_usb2>;
 				phy-names = "usb";
 			};
 
+			utmi_usb2: utmi@5f000 {
+				status = "okay";
+			};
+
 			interrupt-controller@74840 {
 				status = "okay";
 			};
diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 59ca807..c5f3ca1 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -351,6 +351,14 @@
 				status = "disabled";
 			};
 
+			utmi_usb32: utmi@5d000 {
+				compatible = "marvell,armada-3700-utmi-phy";
+				reg = <0x5d000 0x1000>;
+				utmi-port = <UTMI_PHY_TO_USB3_HOST0>;
+				#phy-cells = <0>;
+				status = "disabled";
+			};
+
 			otg_phy: otg@5d800 {
 				compatible = "marvell,armada-3700-otg";
 				reg = <0x5d800 0x100>;
@@ -366,14 +374,6 @@
 				status = "disabled";
 			};
 
-			utmi_usb32: utmi@5d000 {
-				compatible = "marvell,armada-3700-utmi-phy";
-				reg = <0x5d000 0x1000>;
-				utmi-port = <UTMI_PHY_TO_USB3_HOST0>;
-				#phy-cells = <0>;
-				status = "disabled";
-			};
-
 			utmi_usb2: utmi@5f000 {
 				compatible = "marvell,armada-3700-utmi-phy";
 				reg = <0x5f000 0x1000>;
-- 
2.7.4

